### ISSUE TITLE
archive this repo?

### ISSUE THREAD

**Issue by boredsquirrel** [Drakulix/infinity#2]
Thanks for your work! It seems to be continued in another repo now by @ryanabx so I think it would be best to archive this one?

**Comment by Drakulix** [comment]
I am currently still using this one myself, though I would recommend @ryanabx fork. I will archive this eventually and link to that one.

**Comment by ryanabx** [comment]
I don't think this needs to be archived, unless drakulix would like to of course
